### PR TITLE
Merge 7.7 betas into develop

### PR DESCRIPTION
**Pull request by jkmassel**
Merges `7.7` betas into `develop` in advance of the release finalization

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -9,6 +9,10 @@
 - [***] In-Person Payments: Added support for accepting In-Person Card Payments in the US
 - [*] Help & Support: A site's System Status Report can now be found, copied, and shared from Settings > Help & Support > System Status Report [https://github.com/woocommerce/woocommerce-android/pull/4890]
 
+7.6.1
+-----
+- [*] Reverted the support for multi store notifications due to a bug with new order notifications not being received.
```
Comment (AliSoftware) [inline RELEASE-NOTES.txt:14]: This is new 😺 (adding release notes for hotfixes). I like it 👍  even though we need to ensure everyone is clear that those notes won't be included in the PlayStore (hotfix being timely, too late to go thru editorial and translation, yadda yadda). But it's still nice to keep a trace in the repo.

We should probably document that rules better – we already discussed and decided in Slack / async-mtg about what we do hotfix release notes (ie, we don't add them given timing) – but not sure we've shared that widely for all devs to be aware; and that shouldn't prevent us from having a history log in the repo even if the entry is ignored by tooling. 🙃

File: WooCommerce/build.gradle [hunk 2]
```diff
@@ -54,9 +54,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "7.7-rc-2"
+            versionName "7.7-rc-4"
```
Comment (AliSoftware) [inline WooCommerce/build.gradle:57]: Just a note for self that the skip is expected here as this PR merges 2 betas made closely in a row – as confirmed by the commit list/history – so all good there 👍

File: WooCommerce/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -738,6 +738,7 @@
         -->
     <string name="card_reader_manage_card_reader">Manage Card Reader</string>
     <string name="card_reader_purchase_card_reader">Order Card Reader</string>
+    <string name="card_reader_manual_card_reader">Card Reader Manual</string>
```
Comment (AliSoftware) [inline WooCommerce/src/main/res/values/strings.xml:741]: ⚠️ This new string was added at that state of the cycle, i.e. on the `release/*` branch, but not reflected in the frozen `fastlane/resources/values/strings.xml` file, and thus won't be translated in time for 7.7.

I'll let you decide if it's worth including or not; though at that point given the proximity of the end of sprint it's probably too late to get it translated anyway… but just wanted to be sure everyone were aware of this.

**Review by AliSoftware — APPROVED** [review]
Left a warning about a new string introduced in `release/` branch but not in the frozen strings and thus won't be translated.

I'll let you decide what to do with that (replicate the change in the frozen file, or accept that it will only be translated starting 7.8) – which is why I've disabled auto-merge to leave the the opportunity to fix it before merge if you think it's worth it.

Other than that, LGTM 👍

File: WooCommerce/src/main/res/values/strings.xml (hunk 3 above)
Comment (jkmassel) [reply WooCommerce/src/main/res/values/strings.xml:741]: Yeah thanks for the heads up – this was added last-second as a contractual thing, so it's ok that it'll be untranslated for a cycle – thanks for catching it!!

**Pull request merged**